kuma/search/tests/test_utils.py

```diff
@@ -43,6 +43,19 @@ def test_merge_query_param(self):
         eq_(url.merge_query_param('foo', [None]),
             'http://example.com/?foo=&foo=bar&spam=eggs')
 
+    def test_clean_params(self):
+        original = 'http://example.com/?spam='
+        url = QueryURLObject(original)
+
+        eq_(url.clean_params(url.query_dict),
+            {})
+
+        original = 'http://example.com/?spam'
+        url = QueryURLObject(original)
+
+        eq_(url.clean_params(url.query_dict),
+            {})
```

Comment:
This could be in a loop... `for original_url in ('http://example.com/?spam=', 'http://example.com/?spam'): ...`